x11-libs/gtksourceview/gtksourceview3-3.24.0.recipe

```diff
@@ -0,0 +1,101 @@
+SUMMARY="A text widget implementing syntax highlighting and other features"
+DESCRIPTION="GtkSourceView is a GNOME library that extends GtkTextView, the standard GTK \
+widget for multiline text editing. GtkSourceView adds support for syntax \
+highlighting, undo/redo, file loading and saving, search and replace, a \
+completion system, printing, displaying line numbers, and other features \
+typical of a source code editor."
+HOMEPAGE="https://wiki.gnome.org/Projects/GtkSourceView/"
+COPYRIGHT="Christian Hergert
+Sebastien Wilmet"
+LICENSE="GNU LGPL v2.1"
+REVISION="2"
+SOURCE_URI="https://gitlab.gnome.org/GNOME/gtksourceview/-/archive/$portVersion/gtksourceview-$portVersion.tar.gz"
+SOURCE_DIR="gtksourceview-$portVersion"
+CHECKSUM_SHA256="46cd7be87df194473a7ed26af486478f9b3521fda7fb7537f6ad2951ac5eb86d"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	gtksourceview3$secondaryArchSuffix = $portVersion compat >= 3
+	lib:libgtksourceview_3.0$secondaryArchSuffix = $portVersion compat = 3
+	"
+	
+
+REQUIRES="
+	haiku$secondaryArchSuffix	
+	lib:libatk_1.0$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libfribidi$secondaryArchSuffix
+	lib:libgdk_pixbuf_2.0$secondaryArchSuffix
+	lib:libgirepository_1.0$secondaryArchSuffix
+	lib:libgio_2.0$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libgtk_3$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libpango_1.0$secondaryArchSuffix
+	lib:libpangocairo_1.0$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	gtksourceview3${secondaryArchSuffix}_devel = $portVersion compat = 3
+	devel:libgtksourceview_3.0$secondaryArchSuffix = $portVersion compat = 3
+	"
+REQUIRES_devel="
+	gtksourceview3$secondaryArchSuffix == $portVersion base
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgtk_3$secondaryArchSuffix
+	devel:libgirepository_1.0$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+        haiku${secondaryArchSuffix}_devel
+        devel:libatk_1.0$secondaryArchSuffix
+        devel:libcairo$secondaryArchSuffix
+        devel:libfribidi$secondaryArchSuffix
+        devel:libgdk_pixbuf_2.0$secondaryArchSuffix
+        devel:libgtk_3$secondaryArchSuffix
+        devel:libgirepository_1.0$secondaryArchSuffix
+        devel:libglib_2.0$secondaryArchSuffix
+        devel:libintl$secondaryArchSuffix
+        devel:libpango_1.0$secondaryArchSuffix
+        devel:libpangocairo_1.0$secondaryArchSuffix
+        devel:libxml2$secondaryArchSuffix
+        "
+BUILD_PREREQUIRES="
+        autoconf_archive
+        cmd:aclocal
+        cmd:autoconf
+	cmd:libtoolize
```

Comment:
Move libtoolize to proper location (sort alphabeticly) and use: "cmd:libtoolize$secondaryArchSuffix"